CHANGELOG.md

```diff
@@ -1,3 +1,63 @@
+# 84.3.0
+
+This minor release adds dynamic type support to Chips, new theming extensions for Lists, updates to Bottom Drawer elevation shadow. It also includes minior bug fixes and documentation updates to multiple components.
```

Comment:
To clarify: we didn't add Dynamic Type support in general, we added support for the new Material Dynamic Type curves.

"This minor release adds support for new Material Dynamic Type curves to Chips"

Please clarify this below as well.

Reply:
I believe this API has a bug and will be replaced in the next release, which is why we skipped this feature in this release.